log-generator/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+module.exports = {
+  extends: '@newrelic',
+  overrides: [
```

Comment:
I wouldn't turn off this rule.  We try to require having the copyright header on every file

Reply:
Agreed. If that file is failing... you probably just need to remove the newline between the `#!` and the `/*`

Reply:
Oh, look at that. I was following [this example](https://github.com/newrelic/node-newrelic/blob/main/.eslintrc.js#L18), but you're right, I can remove the blank line after the shebang instead.